scipy/optimize/_differentialevolution.py

```diff
@@ -20,9 +21,10 @@
 _MACHEPS = np.finfo(np.float64).eps
 
 
+@_transition_to_rng("seed", position_num=10)
```

Comment:
```suggestion
@_transition_to_rng("seed", position_num=9)
```
Right? Looks like `seed`/`rng` was/is at position 9 (0-indexed).

Reply:
Good catch. How on earth did the tests pass then.